engine/class_modules/paladin/sc_paladin_retribution.cpp

```diff
@@ -390,14 +397,27 @@ struct blade_of_justice_t : public paladin_melee_attack_t
 
     if ( p()->talents.expurgation->ok() )
     {
-      expurgation->set_target( state->target );
-      expurgation->execute();
+      p()->active.expurgation->target = state->target;
```

Comment:
If we're no longer going to use the direct child expurgation instance, can we not initialize it for each blade_of_justice_t initialization?

Reply:
Fixed in https://github.com/simulationcraft/simc/pull/8169/commits/8f40d6bcddea9e7cb62dfe72e2c43c29b4d07553